scripts/template_upload_rhevm.py

```diff
@@ -243,7 +243,7 @@ def make_vm_from_template(mgmt, stream, cfme_data, cluster, temp_template_name,
         template = mgmt.get_template(temp_template_name)
         vm = template.deploy(
             vm_name=temp_vm_name,
-            cluster=cluster,
+            cluster_name=cluster,
```

Comment:
do we not need to mention `storage_domain` in here? RHV would by default deploy template on same domain where the template resides.

Reply:
This script is deprecated, will be removing it in another PR. Changes you want to pay attention to are in cfme/utils/templates/